monkestation/code/modules/donator/code/datum/store_items.dm

```diff
@@ -40,3 +40,33 @@
 	name = "Syndie Pajamas"
 	item_path = /obj/item/clothing/under/costume/syndie_pajamas
 	item_cost = 4000
+
+//riddler
+/datum/store_item/suit/dark_hos
+	name = "Dark HoS Trenchcoat"
+	item_path = /obj/item/clothing/suit/costume/dark_hos
+	item_cost = 7000 //you gotta pay for coolness
```

Comment:
dont add these, as it means non patrons can buy it and than can't actually use it